tests/phpunit/trait-test-upgrades.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Try to convert this trait to a base abstract class that extends `Elementor_Test_Base` and both core and pro Upgrades classes inherit it